web/handler/setup.go

```
@@ -0,0 +1,137 @@
+		serverError(w, err)
+		return
+	}
+	settings.Name = setupForm.ForumName
+	settings.URL = setupForm.URL
+	settings.Lang = setupForm.Lang
+	if err := h.storage.UpdateSettings(settings); err != nil {
+		serverError(w, err)
+		return
+	}
+
+	if err := h.seedSampleContent(adminID); err != nil {
```

golangci-lint (2.12.2)

[error] Function `seedSampleContent->CreateTopic` should pass the context parameter

(contextcheck)